optionals/compat_rhs_usf3/CfgWeapons.hpp

```diff
@@ -112,15 +115,31 @@ class CfgWeapons
     class rhs_weap_mk18: rhs_weap_m4a1 {
         ACE_barrelTwist=177.8;
         ACE_barrelLength=261.62;
+        ACE_Overheating_Dispersion[] = {0, 0.001, 0.002, 0.004};
+        ACE_Overheating_SlowdownFactor[] = {1, 1, 1, 0.9};
+        ACE_Overheating_JamChance[] = {0, 0.0003, 0.0015, 0.0075};
     };
     class rhs_weap_m16a4: rhs_weap_m4_Base {
-        ACE_barrelTwist=177.8;
-        ACE_barrelLength=508.0;
-    };
+        ACE_barrelTwist = 177.8;
+        ACE_barrelLength = 508.0;
+    };
+	class rhs_weap_lmg_minimipara;	
+    class rhs_weap_lmg_minimi_railed : rhs_weap_lmg_minimipara {
+        ACE_barrelLength = 465.0;
+        ACE_barrelTwist = 177.8;
+		ACE_Overheating_allowSwapBarrel = 1;
+        ACE_Overheating_Dispersion[] = {0, -0.001, 0.001, 0.003};
+        ACE_Overheating_SlowdownFactor[] = {1, 1, 1, 0.9};
+        ACE_Overheating_JamChance[] = {0, 0.0003, 0.0015, 0.0075};
+	};
     class rhs_weap_m240_base;
     class rhs_weap_m240B: rhs_weap_m240_base {
-        ACE_barrelTwist=304.8;
-        ACE_barrelLength=629.92;
+        ACE_barrelTwist = 304.8;
+        ACE_barrelLength = 629.92;
+        ACE_Overheating_allowSwapBarrel = 1;
+        ACE_Overheating_Dispersion[] = {0, -0.001, 0.001, 0.003};
+        ACE_Overheating_SlowdownFactor[] = {1, 1, 1, 0.9};
+        ACE_Overheating_JamChance[] = {0, 0.0003, 0.0015, 0.0075};
```

Comment:
Does anyone know if the overheating config is setup properly. I put 500 rounds through the gun and it didnt get hot at all.